homeassistant/components/sensor/geo_rss_events.py

```diff
@@ -87,14 +81,18 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 class GeoRssServiceSensor(Entity):
     """Representation of a Sensor."""
 
-    def __init__(self, category, data, service_name, unit_of_measurement):
+    def __init__(self, home_coordinates, url, radius, category, service_name,
+                 unit_of_measurement):
         """Initialize the sensor."""
         self._category = category
-        self._data = data
         self._service_name = service_name
         self._state = STATE_UNKNOWN
```

Comment:
Init state as `None`.

Reply:
As above.

Reply:
We use `None` to represent unknown state, so it won't break anything.